app/services/generate_new_week_assignments.rb

```diff
@@ -16,13 +16,19 @@ def call
     def generate_last_assignment_list
       @last_assignment_list = []
 
-      Employee.all.each do |employee|
+      Employee.find_each do |employee|
         last_employee_assignment = Assignment.where(employee_id: employee.id).order(:week_id).last
 
         if last_employee_assignment.present?
           @last_assignment_list.push(last_employee_assignment)
         else
-          #new employee who haven't been assigned yet dealt with here
+          temp_assignment = Assignment.create(week: @week, employee: employee)
+
+          last_employee_assignment = Assignment.where(employee_id: employee.id).order(:week_id).last
+
+          @last_assignment_list.push(last_employee_assignment)
+
+          temp_assignment.destroy
```

Comment:
This is a very brute force way to fix the no assignments for the employee. I don't plan on using it